compute_tools/src/http/api.rs

```diff
@@ -98,6 +100,38 @@ async fn routes(req: Request<Body>, compute: &Arc<ComputeNode>) -> Response<Body
             }
         }
 
+        (&Method::POST, "/extensions") => {
+            info!("serving /extensions POST request");
```

Comment:
I think we are moving to make these `debug!`

Reply:
I was going off @lassizci's comment here: https://github.com/neondatabase/neon/pull/9375#discussion_r1800925713. But perhaps the context is just completely different.

Reply:
Yes, for metrics it's kinda all computes every dozen of seconds or so. Here, it is a small subset that will use Neon authorize

I think I was still a bit wrong saying that it's only once per DB. It will install extension once per DB, yes, but I guess local proxy doesn't know that it's installed, so will it call it every time we have a new connection?

Probably, Jere or Conrad can elaborate how often it will be called. Considering that only a few users will use it at start, I'd still probably keep the info for the ease of debugging, but do not have a strong opinion

Reply:
> install the extension once per database per boot (then caching that it's installed)
This is how it should work.

We discussed this with @lassizci that we can always drop this to `debug` if this ends up being too noisy.